app/livechat/client/views/app/tabbar/visitorForward.js

```diff
@@ -63,17 +68,42 @@ Template.visitorForward.events({
 			transferData.departmentId = instance.find('#forwardDepartment').value;
 		}
 
-		Meteor.call('livechat:transfer', transferData, (error, result) => {
-			if (error) {
-				toastr.error(t(error.error));
-			} else if (result) {
-				this.save();
-				toastr.success(t('Transferred'));
-				FlowRouter.go('/');
-			} else {
-				toastr.warning(t('No_available_agents_to_transfer'));
-			}
-		});
+		// Check for settings whether forward is enabled or not.
+
+		if (!settings.get('Livechat_ask_for_forward')) {
+			// If asking for forward permission disabled forward normally
+			Meteor.call('livechat:transfer', transferData, (error, result) => {
+				if (error) {
+					toastr.error(t(error.error));
+				} else if (result) {
+					this.save();
+					toastr.success(t('Transferred'));
+					FlowRouter.go('/');
+				} else {
+					toastr.warning(t('No_available_agents_to_transfer'));
+				}
+			});
+			return;
+		}
+
+		const timeoutAgent = Math.abs(settings.get('Livechat_forward_timeout_second') * 1000);
+		const userdeny = Meteor.users.findOne(transferData.userId);
```

Comment:
Please, use `camelCase` approach when naming `const`.
Thanks.
```suggestion
		const userDeny = Meteor.users.findOne(transferData.userId);
```